lib/pages/home_page.dart

```diff
@@ -13,54 +14,131 @@ class HomePage extends StatefulWidget {
 }
 
 class _HomePageState extends State<HomePage> {
-  LocationPermission? gpsPermission;
-  Position? position;
+  LocationPermission? gps = LocationPermission.denied;
+  Position? pos;
 
-  @override
-  void initState() {
-    foo();
+  Future<void> foo() async {
```

Comment:
Ty jména funkcí a metod se pokud možno snaž držet tak aby to vypovídalo o tom, co to dělá